src/i18n/resources/fr.js

```diff
@@ -480,6 +480,22 @@ L'équipe de FRA
     invitationLink: "Lien d'invitation",
     sendInvitation: "Envoyer un email d'invitation",
     inviteAgain: 'Inviter de nouveau',
+    invitationAcceptedNotification: {
```

Comment:
i would introduce new file and/or folder (you can see what is better) for emails:
en/emails/<email>
or
en/email.js

used as

`email.userInvitation.acceptedNotification: { ... }`